docs/general_config.md

```diff
@@ -145,6 +145,8 @@ allSessionsCompleteFeedback: "All Sessions Complete!",
 |`horizontalFieldOfView`    |°      | The (horizontal) field of view for the user's display, to get the vertical FoV multiply this by `1 / your display's aspect ratio` (9/16 for common FHD, or 1920x1080)|
 |`frameDelay`               |frames | An (integer) count of frames to delay to control latency           |
 |`frameRate`                |fps/Hz | The (target) frame rate of the display (constant for a given session) for more info see the [Frame Rate Modes section](#Frame-Rate-Modes) below.|
+|`frameTimeArray`           |`Array<float>`| An array of frame times to use instead of `frameRate` if populated, otherwise ignored. |
```

Comment:
Would be good to include the units for these frame times here. Could do it in the units field as `Array<float>` s or `Array<`s`>`, or just specify it in the description. I assume these times are specified in seconds?